src/instructlab/model/serve_backend.py

```diff
@@ -80,11 +114,25 @@ def serve_backend(
     elif backend == backends.VLLM:
         # First Party
         from instructlab.cli.model.serve import warn_for_unsupported_backend_param
-        from instructlab.utils import contains_argument
 
         warn_for_unsupported_backend_param(ctx)
 
         ctx.obj.config.serve.vllm.vllm_args = ctx.obj.config.serve.vllm.vllm_args or []
+
+        # Fix ValueError: Too large swap space. 16.00 GiB out of the 15.01 GiB total CPU memory is allocated for the swap space.
+        check_size, effective_gpus, available_gpus = check_tensor_parallel_size(
+            ctx.obj.config.serve.vllm.vllm_args
+        )
+        if check_size:
+            # Third Party
+            import click
```

Comment:
Would click not always be imported already?  Any reason not to put at the top of the file?

Reply:
I think just call it by this check, so just put it in the check. It is ok to put it at the top of the file.